### ISSUE TITLE
Integrate Travis CI build and unit tests

### ISSUE THREAD

**Issue by peter279k** [NorseBlue/parentity#1]
As title, to make this package reliable, we should consider using Travis CI build and PHPUnit to accomplish this issue.

Once this issue is accepted, I glad I can work on this :).

**Comment by axelitus** [comment]
I agree this needs to conform to the configuration of the latest packages (e.g. [norse-blue/php-handy-properties](https://github.com/norse-blue/php-handy-properties)), though at this moment this package is not a priority and is still a proof of concept (it also needs to be tested and updated to support the latest Laravel versions).
